client/common/termutils.js

```diff
@@ -22,7 +22,13 @@ export async function getterm(termid, dslabel = null, genome = null) {
 	}
 	if (!dslabel) throw 'getterm: dslabel missing'
 	if (!genome) throw 'getterm: genome missing'
-	const data = await dofetch3(`termdb?dslabel=${dslabel}&genome=${genome}&gettermbyid=${termid}`)
+	const body = {
```

Comment:
i don't see `getterm()` exported from here used anywhere. it duplicates vocab api. can try to delete this function and test